README.md

```diff
@@ -2,28 +2,66 @@
 
 A Python module for communication with [PEKAT VISION](https://www.pekatvision.com/products/software/).
 
+Full SDK documentation available here: <https://pekat-vision.github.io/pekat-vision-sdk-python>
+
 ## Installation
 
-Type `pip install "pekat-vision-sdk[opencv]"` into your terminal.
+Type `pip install "pekat-vision-sdk"` into your terminal.
+
+Installing with `pip install "pekat-vision-sdk[opencv]"` also installs [`opencv`](https://pypi.org/project/opencv-python/).
 
 ## Example
 
+### Creating the analyzer
+
 ```python
-import cv2
 from PekatVisionSDK import Instance
 
+# Start a project locally
+
+p_local = Instance("~/PekatVisionProjects/my_project", port=8100, host="0.0.0.0")
```

Comment:
```suggestion
# Start a project locally (host 0.0.0.0 - listen on all interfaces)

p_local = Instance("~/PekatVisionProjects/my_project", port=8100, host="0.0.0.0")
```
I would add this to people who would copy this line without any knowledge what 0.0.0.0 means even that it is stated in the class docstring.